dangerfile.js

```diff
@@ -38,6 +38,16 @@ if (packagesModified && !unreleasedModified && !declaredTrivial) {
   warn("One or more packages have changed, but `UNRELEASED.md` wasn't updated.");
 }
 
+// If any files were created, the BpkComponentUsageDetector should have been updated.
```

Comment:
added a danger check to warn when files for components were created to update the detector as well